Integrations/integration-EWSv2.yml

```diff
@@ -185,6 +185,11 @@ script:
             return dict((k, v) for k, v in d.items() if v is not None)
         return d
 
+    def get_attachment_name(attachment_name):
+        if not attachment_name:
```

Comment:
@BenJoParadise if the attachment name would be 0 we will return 'demisto_untitled_attachment' instead of 0. Please fix that scenario if relevant.